### ISSUE TITLE
collections.Iterator incompatibility?

### ISSUE THREAD

**Issue by devzzzero** [0rpc/zerorpc-python#266]
Hi, I am playing with python 3.10

and here is my zerorpc version
```
$ pip show zerorpc
Name: zerorpc
Version: 0.6.3
Summary: zerorpc is a flexible RPC based on zeromq.
Home-page: https://github.com/0rpc/zerorpc-python
Author: François-Xavier Bourlet <bombela+zerorpc@gmail.com>.
Author-email: UNKNOWN
License: MIT
Location: /home/ai/MiniConda3/envs/rag/lib/python3.10/site-packages
Requires: future, gevent, msgpack, pyzmq
Required-by: 
```
The simple test in the [readme](https://github.com/0rpc/zerorpc-python/tree/master?tab=readme-ov-file#exposing-a-zeroservice-programmatically)

is barfing on the zerorpc client call side with

AttributeError: module 'collections' has no attribute 'Iterator'

Was this meant to be `collections.abc.Iterator` ?

**Comment by devzzzero** [comment]
TOP: woops. this is a duplicate of #250
